transforms/packaging/python/Makefile

```diff
@@ -0,0 +1,42 @@
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below.
+
+# $(REPOROOT)/.make.versions file contains the versions
+
+include $(REPOROOT)/transforms/.make.transforms
+include ../.make.packaging
+
+PACKAGING_RUN_TIME=python
+DPK_TRNASFORM_REV=0.2.1.dev1
```

Comment:
This should come from DPK_LIB_VERSION and perhaps be defined in .make.versions at the top of the repo.